benches/bench.rs

```diff
@@ -307,3 +311,40 @@ fn http_concurrent_conn_calls(rt: &TokioRuntime, crit: &mut Criterion, url: &str
 	}
 	group.finish();
 }
+
+/// Bench `round_trip` with different header sizes.
+fn http_custom_headers_round_trip(
+	rt: &TokioRuntime,
+	crit: &mut Criterion,
+	url: &str,
+	name: &str,
+	request: RequestType,
+) {
+	for header_size in [1 * KIB, 2 * KIB, 8 * KIB] {
```

Comment:
```suggestion
	for header_size in [KIB, 2 * KIB, 8 * KIB] {
```
Does it make sense to just have KIB instead of multiplying it by 1?

Reply:
That also makes sense for me! Thanks!